### PR TITLE
VZ-5008: Change Fluentd is use the verrazzano-es-internal secret

### PR DESCRIPTION
**Pull request by bobdonat**
# Description

This pull request changes the Fluentd configuration to use the verrazzano-es-internal secret instead of the verrazzano secret.  In addition, fixed a bug to change the order of values files supplied to Helm.  Previously, the order was overrides and then values from 'helm get valued.  That order is now reversed.

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: application-operator/mcagent/mcagent.go [hunk 1]
```diff
@@ -331,7 +331,7 @@ func (s *Syncer) updateLoggingDaemonSet(regSecret corev1.Secret, ds *appsv1.Daem
 const (
 	defaultClusterName   = constants.DefaultClusterName
 	defaultElasticURL    = "http://verrazzano-authproxy-elasticsearch:8775"
-	defaultSecretName    = "verrazzano"
+	defaultSecretName    = "verrazzano-es-internal" //nolint:gosec //#gosec G101
```
Comment (pfmackin) [inline application-operator/mcagent/mcagent.go:334]: what is this //nolin.. comment

Comment (bobdonat) [reply application-operator/mcagent/mcagent.go:334]: That prevents the linter failing with 
```
G101: Potential hardcoded credentials (gosec)
17:38:44  	defaultSecretName    = "verrazzano-es-internal"
```

**Review by wmhopkins — APPROVED** [review]
lgtm

**Review by wmhopkins — APPROVED** [review]
Looks good.

**Pull request merged**